### PR TITLE
0001 - Create a CONTRIBUTING file

### PR DESCRIPTION
**Pull request by KrashLeviathan**
### Problem / Feature
We needed a CONTRIBUTING.md file to outline the process for contributing to the repo.

### Solution
Made the CONTRIBUTING file, and also made a refactor for one of the folder names.

### What To Test
Did I miss anything we need in there?

### Reviewers

@regalcat 

FYI: @KrashLeviathan @csteamengine

### CODE REVIEW

**Comment by KrashLeviathan** [comment]
Not sure if that tutorial I linked to is all that great, I just needed something to put in there for now. If we find a better one later (or when we make our own!) we can change the link.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+**Initial steps:**
+
+- Setup a GitHub account
+- Fork the club-website repo
+- Clone it to your desktop
+- You're ready to go!
+
+Then just head to the [Issues](https://github.com/ISU-WebDevClub/club-website/issues) page to find a fun feature or bug to tackle. If you have problems or questions, feel free to reach out to one of the other club members, and they should be able to give you a hand getting started.
+
+## Issue Tracking
```
Comment (MichaelRupertDev) [inline CONTRIBUTING.md]: Add issue tracking template?

**Comment by regalcat** [comment]
Looks good
+1

File: CONTRIBUTING.md (hunk 1 above)
Comment (KrashLeviathan) [inline CONTRIBUTING.md]: That sounds like a problem for future Michael and future Nathan! Haha. We could update it later as we see fit, but for now I just want to get this thing posted so newcomers have a sense of direction when they look at the repo for the first time.

**Pull request merged**